mh/constants.py

```diff
@@ -164,13 +165,43 @@ def make_binary_npc_greeters(is_jap=False):
     US_OFFSET = 0x180
     JP_OFFSET = 0x100
     offset = JP_OFFSET if is_jap else US_OFFSET
+
+
+    if is_jhen_active():
+        tool_shop_str = b"Half-off sale!"
+        material_shop_str = b"Half-off sale!"
+        event_quests_str = "The Festival of Fear\n"
+    else:
+        tool_shop_str = b"Prices are normal."
+        material_shop_str = b"Prices are normal."
+        event_quests_str = ""
+
+    event_quests_str += get_quest_name(QUEST_EVENT_JUMP_FOUR_JAGGI) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_BLOOD_SPORT) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_MERCY_MISSION) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_THE_PHANTOM_URAGAAN) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_WORLD_EATER) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_WHERE_GODS_FEAR_TO_TREAD) + '\n'
+    event_quests_str += get_quest_name(QUEST_EVENT_GREEN_EGGS)
+
+    guildmaster_str = (
```

Comment:
The `_str` suffix in the variable names should be removed. It's superfluous and doesn't reflect that these should be `bytes`. Moreover, the function is short and mostly self-contained with an explicit name `npc_greeters` so I don't think more context is needed to understand the variables purpose.

Reply:
Changed